nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestProbesObject.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Thomas Roger
+ *     Nour Al Kotob
+ */
+
+package org.nuxeo.rest.management;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.Test;
+import org.nuxeo.ecm.core.io.marshallers.json.JsonAssert;
+import org.nuxeo.ecm.core.management.api.ProbeManager;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.test.runner.Deploy;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * @since 11.1
+ */
+@Deploy("org.nuxeo.ecm.core.management")
+public class TestProbesObject extends ManagementBaseTest {
```

Comment:
Is it possible to add assertions on other fields in response?

Reply:
Yes in the `ProbeInfoWriter` you can see all the info that is in the response, however, some will vary depending on the platform and context

Reply:
I think we could add assert them if we can control them in tests.

Reply:
to complete the above comment, you can try to use Mockito to mock the manager prob if needed to add more tests 
have look [here](https://github.com/nuxeo/nuxeo/blob/7e61b0f0706e2974ce5e157aefb61eea1d17eee2/nuxeo-core/nuxeo-core-api/src/test/java/org/nuxeo/ecm/core/api/security/TestACP.java#L59) to have an idea. 
wdyt @ataillefer